maintenance_equipment_sequence/models/maintenance.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class MaintenanceEquipmentCategory(models.Model):
+    _inherit = 'maintenance.equipment.category'
+
+    code = fields.Char(
+        string='Sequence Prefix',
+        size=5,
+        help="The equipment's sequence will be created using this prefix."
+    )
```

Comment:
Reproduced in Runbot:

Open an existing category that has already a sequence attached: the code value here is not displayed.

Could it be confusing?